app/models/workflow.rb

```diff
@@ -32,7 +32,7 @@ class Workflow < ActiveRecord::Base
   # TODO: remove this association from the cache key
   cache_by_resource_method :subjects_count, :finished?
 
-  enum subject_selection_strategy: [:default, :cellect, :designator]
+  enum subject_selection_strategy: [:default, :cellect, :designator, :builtin]
 
   scope :using_cellect, -> { where(subject_selection_strategy: subject_selection_strategies[:cellect]) }
```

Comment:
do you want to modify this scope to pickup non cellect enum targets?

Reply:
The only thing that used this was the Cellect controller, and i didn't want that to return non-Cellect workflows. 

But given that you've now confirmed that nothing uses this controller yet, I can remove this completely and let the cellect controller take a `strategy` param. (And rename that controller)